### PR TITLE
Initial tezos-ci deployer

### PR DESCRIPTION
**Pull request by tmcgilchrist**

### CODE REVIEW

File: src/pipeline.ml [hunk 1]
```diff
@@ -214,5 +218,9 @@ let v ?app ?notify:channel ?filter ~sched ~staging_auth () =
     ];
     ocaml_bench, "sandmark-nightly", [
       docker "Dockerfile" ["main", "ocurrent/sandmark-nightly:live", [`Ci3 "sandmark_sandmark"]]
+    ];
+
+    tarides, "tezos-ci", [
+      docker "Dockerfile" ["live", "ocurrent/tezos-ci-service:live", [`Tezos "tezos-ci_ci"]]
```
Comment (tmcgilchrist) [inline src/pipeline.ml:224]: Pushing to docker hub as `ocurrent/tezos-ci-service` perhaps it should go somewhere else?

File: config/docker/contexts/meta/afefb91b38561ce4c5ed9f03d31ba2e93d8d878f6d2817a2851641fbdd290343/meta.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"Name":"tezos","Metadata":{"StackOrchestrator":"swarm"},"Endpoints":{"docker":{"Host":"ssh://root@tezos-ci.ocamllabs.io","SkipTLSVerify":false}}}
```
Comment (tmcgilchrist) [inline config/docker/contexts/meta/afefb91b38561ce4c5ed9f03d31ba2e93d8d878f6d2817a2851641fbdd290343/meta.json:1]: `Host` might not be the final place this service runs.

File: src/pipeline.ml (hunk 1 above)
Comment (tmcgilchrist) [reply src/pipeline.ml]: Pushing to `ocurrent/tezos-ci` for now

**Review by mtelvers — APPROVED** [review]
Looks good.

**Pull request merged**